inlong-sort/sort-flink/sort-flink-v1.15/sort-connectors/starrocks/pom.xml

```diff
@@ -45,6 +45,10 @@
             <artifactId>flink-connector-starrocks</artifactId>
             <version>${starrocks-connector.version}_flink-${flink.starrocks.version}</version>
         </dependency>
+        <dependency>
```

Comment:
It is recommend to use jackson or google json.

Reply:
Thanks, it's tested in official starrocks connector, will do in the next version